### ISSUE TITLE
identical result from GLM and FarmCPU

### ISSUE THREAD

**Issue by binlu1981** [xiaolei-lab/rMVP#72]
Hi @XiaoleiLiuBio @YinLiLin @hyacz ,
I got exactly the same p-values from GLM and FarmCPU, but different results from  MLM method. The rMVP version is 1.06 and set method.bin="static". Is there a bug or some problems?

Best 
Bin

**Comment by XiaoleiLiuBio** [comment]
Hi Bin,

Thank you for using rMVP. If there is no signal passed the Bonferroni cutoff in GLM, FarmCPU will stop and output the same results as GLM.

Best,
Xiaolei


---------------- 

---------- Origin message ----------
>From："Bin Lu" ***@***.***>
>To："xiaolei-lab/rMVP" ***@***.***>
>Subject：[xiaolei-lab/rMVP] identical result from GLM and FarmCPU (Issue #72)
>Date：2022-02-08 14:41:32Hi @XiaoleiLiuBio @YinLiLin @hyacz ,
I got exactly the same p-values from GLM and FarmCPU, but different results from MLM method. The rMVP version is 1.06 and set method.bin="static". Is there a bug or some problems?

Best
Bin

--
Reply to this email directly or view it on GitHub:
https://github.com/xiaolei-lab/rMVP/issues/72

**Comment by binlu1981** [comment]
Hi @XiaoleiLiuBio ,
Thanks for quick reply. I have nearly 30 million snps, so the correction cutoff seems too stringent for the data. How can I make FarmCPU work? Should I set p.threshold higher or change some other parameter? 

Best
Bin

**Comment by XiaoleiLiuBio** [comment]
Yes, you can change the p.threshold to an appropriate cutoff to make FarmCPU work but should be very careful. The default of p.threshold is set to Bonferroni cutoff.


---------------- 

---------- Origin message ----------
>From："Bin Lu" ***@***.***>
>To："xiaolei-lab/rMVP" ***@***.***>
>Subject：Re: [xiaolei-lab/rMVP] identical result from GLM and FarmCPU (Issue #72)
>Date：2022-02-08 22:30:11Hi @XiaoleiLiuBio ,
Thanks for quick reply. I have nearly 30 million snps, so the correction cutoff seems too stringent for the data. How can I make FarmCPU work? Should I set p.threshold higher or change some other parameter?

Best
Bin

**Comment by binlu1981** [comment]
Thanks Xiaolei. One more question. Which method for selecting the most appropriate bins is preffered, ’static’, ’EMMA’ or ’FaST-LMM’?

Best 
Bin
